tests/datasets/test_splits.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from math import floor, isclose
+from typing import Any, Dict, List, Sequence, Tuple, Union
+
+import pytest
+from rasterio.crs import CRS
+
+from torchgeo.datasets import GeoDataset
+from torchgeo.datasets.splits import (
+    random_bbox_assignment,
+    random_bbox_splitting,
+    random_grid_cell_assignment,
+    roi_split,
+    time_series_split,
+)
+from torchgeo.datasets.utils import BoundingBox
```

Comment:
Could import all of these from `torchgeo.datasets` if you want to test that the import alias works